ansible/vars/all.yml

```diff
@@ -0,0 +1,67 @@
+---
+
+gcp_ssh_key_file: "{{ lookup('env', 'GCP_SSH_KEY_FILE', default='~/.ssh/google_compute_engine') }}"
+
+ansible_ssh_private_key_file: "{{ gcp_ssh_key_file }}"
+
+current_user: "{{ lookup('env', 'USER') }}"
+job_id: "{{ lookup('env', 'JOB_ID', default=current_user) }}"
+test_platform: "{{ lookup('env', 'VM_TYPE', default='rhel') }}"
+collector_root: "{{ playbook_dir }}/.."
```

Comment:
This is just me talking to myself, but it'd be nice if we could use something like `git` to determine the root of the directory, rather than using relative paths.